.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+### Contribution Best Practices
+
+**Feature Requests and Bug Reports**
```

Comment:
This section is not useful

Reply:
I don't see any of the guidelines specified [here](https://github.com/HTTP-APIs/hydrus/wiki/Starting-Material#general-guidelines-to-contributing) mentioned.